src/System.Private.CoreLib/shared/System/IO/Stream.cs

```diff
@@ -189,6 +189,72 @@ private int GetCopyBufferSize()
             return bufferSize;
         }
 
+        public virtual void CopyTo(ReadOnlySpanAction<byte, object?> callback, object? state, int bufferSize)
+        {
+            if (callback == null) throw new ArgumentNullException(nameof(callback));
+
+            CopyTo(new WriteCallbackStream(callback, state), bufferSize);
+        }
+
+        public virtual Task CopyToAsync(Func<ReadOnlyMemory<byte>, object?, CancellationToken, ValueTask> callback, object? state, int bufferSize, CancellationToken cancellationToken)
```

Comment:
Shouldn't the return value be a `ValueTask` also like the callback as this is a new method?

Reply:
https://github.com/dotnet/coreclr/pull/27639#discussion_r341882644